src/components/stripe-verify.tsx

```
@@ -0,0 +1,9 @@
+import { Button, Link } from '@payloadcms/ui'
+
+export const StripeVerify = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)